src/test/java/com/example/java/test/junior/developer/QuestionServiceTest.java

```diff
@@ -0,0 +1,107 @@
+package com.example.java.test.junior.developer;
+
+import com.example.java.test.junior.developer.dto.QuestionDto;
+import com.example.java.test.junior.developer.mapper.QuestionMapper;
+import com.example.java.test.junior.developer.model.Question;
+import com.example.java.test.junior.developer.repository.QuestionRepository;
+import com.example.java.test.junior.developer.service.QuestionService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Please, we use JUnit5 on the project - it should be `@ExtendWith(MockitoExtention.class)`